client/landing/stepper/declarative-flow/internals/steps-repository/playground/lib/import-playground.ts

```diff
@@ -0,0 +1,39 @@
+import { zipWpContent, type PlaygroundClient } from '@wp-playground/client';
+import { uploadExportFile } from 'calypso/state/imports/actions';
+
+export type ImportStatus =
```

Comment:
We can use functions found here if needed https://github.com/Automattic/wp-calypso/blob/trunk/client/state/imports/api.js

Reply:
I forgot to remove this, it's not used anymore